README.md

```diff
@@ -1,2 +1,13 @@
 # HW01
 Edit the readme
+
+# Yi-wen Wang
+
+Yi-wen Wang a post-doc in University of Chicago. Her research interest lies on **devlopmental neuroscience**. Her educational background is list as follows: 
+  * Taipei medical University    BS
+  * University of Penssylvania   MA
+  * University Of Chicago       PhD
+She is corrently working in Ellie Heckscher's lab (https://voices.uchicago.edu/heckscherlab/), studying how stem cell lineage affect neuronal wiring in Drosophila. 
+
+
+![Alt text](/Users/pc/Desktop/test.jpg)
```

Comment:
It might be nice to use an image hosted on the web somewhere so that we can all see it, instead of using one uploaded from your own computer :)